__tests__/app.test.js

```diff
@@ -42,3 +43,29 @@ describe("GET /api/topics", () => {
     })
   })
 })
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with a single article object to the client", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then((response) => {
+      expect(typeof response.body.article.article_id).toBe('number');
+      expect(typeof response.body.article.author).toBe('string');
+      expect(typeof response.body.article.title).toBe('string');
+      expect(typeof response.body.article.body).toBe('string');
+      expect(typeof response.body.article.topic).toBe('string');
+      expect(typeof response.body.article.created_at).toBe('string');
+      expect(typeof response.body.article.votes).toBe('number');
+      expect(typeof response.body.article.article_img_url).toBe('string');
+    })
+  })
+  test("404: Responds with error message if the id is invalid", () => {
+    return request(app)
+    .get("/api/articles/99987")
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Not found")
+    })
+  })
+})
```

Comment:
I like this test, however, the description mentions an invalid ID, this ID is valid but it just doesn't exist yet. So the description should reflect that.

You should also do a test to check an invalid test such as, "/api/articles/not-an-id".